sdk/personalizer/Azure.AI.Personalizer/src/Models/ClientConfigurationRestClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.Personalizer.Models
+{
+    internal class ClientConfigurationRestClient
```

Comment:
Why isn't this class in the generated folder? Rest clients should not be hand written, only extended.

Reply:
As client Post is not part of swagger because of which it's not part of generated. So implemented it outside generated to get client config details